src/main/resources/export/java/Command-variable-setting.java

```diff
@@ -2,4 +2,7 @@
 #set($params = $command.getProperty("Parameters").getValue())
 #foreach($param in $params)
         m_$param.getName() = $param.getName();
-#end
\ No newline at end of file
+#end
+#if($command.getProperty("DoubleSupplier 1").getValue() != "None")m_doublesupplier1 = doublesupplier1; #end
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [ES: Comparison of String objects using == or != (ES_COMPARING_STRINGS_WITH_EQ)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq).
This code compares java.lang.String objects for reference equality using the == or != operators. Unless both strings are either constants in a source file, or have been interned using the String.intern() method, the same string value may be represented by two different String objects. Consider using the equals(Object) method instead.